data/json/move_modes.json

```
@@ -52,5 +52,27 @@
     "sound_multiplier": 0.5,
     "move_speed_multiplier": 0.5,
     "swim_speed_mod": 50
+  },
+  {
+    "type": "movement_mode",
+    "id": "jogging",
+    "character": "j",
+    "panel_char": "J",
+    "name": "jogging",
+    "panel_color": "yellow",
+    "symbol_color": "yellow",
+    "exertion_level": "BRISK_EXERCISE",
+    "change_good_none": "You start jogging.",
+    "change_good_animal": "You spur your steed to go faster.",
+    "change_good_mech": "You increase the power of your mech's leg servos.",
+    "change_bad_none": "You're too tired to jog.",
+    "change_bad_animal": "Your steed is too tired to go faster.",
+    "change_bad_mech": "Your mech's leg servos are unable to operate faster.",
+    "move_type": "jogging",
```

I don't think this needs a new move mode type, jogging fits perfectly well as a `running` move_mode_type.
```suggestion
    "move_type": "running",
```